sklearn/preprocessing/tests/test_function_transformer.py

```diff
@@ -0,0 +1,83 @@
+from nose.tools import assert_equal
+import numpy as np
+
+from ..function_transformer import FunctionTransformer
```

Comment:
Test should use public import. In fact, I think the module should be called `_function_transformer` so people don't import it directly.